manifests/gcp_marketplace/schema.yaml

```diff
@@ -122,8 +122,16 @@ properties:
       Using CloudSQL and GCS provides better reliability and performance,
       as well as features such as data backup and usage monitoring.
       This is the recommended option especially for production scenarios.
-      If false, the data will be stored in GCE Persistent Disk.
+      If false, the data will be stored in GCE Persistent Disks.
     default: false
+  managedstorage.gcsBucketName:
+    type: string
+    title: Artifact storage GCS bucket (Managed storage only)
+    description: |-
+      This field must be specified if choose to use managed storage.
+      Provide the Google Cloud Storage (GCS) bucket name for an existing bucket.
```

Comment:
Current behavior is that, api server auto creates a bucket with this name if it doesn't exist.

I'm thinking we should add a flag to disable that behavior for managed storage option (very low effort). Just want to know your thoughts on this UX first?

Reply:
Documented current behavior to minimize efforts.